plugins/vpp/ifplugin/data_resync.go

```diff
@@ -464,6 +464,7 @@ func (plugin *NatConfigurator) ResyncDNat(nbDNatConfig []*nat.Nat44DNat_DNatConf
 	// Correlate with existing config
 	for _, nbDNat := range nbDNatConfig {
 		for _, vppDNat := range vppDNatCfg.DnatConfigs {
+			plugin.log.Infof("nbdnatlabel %v , vpp %v", nbDNat.Label, vppDNat.Label)
```

Comment:
Should probably be `Debugf`.

Reply:
Oh, this should not be in this diff at all, oops